test/ui-testing/pick-lists.js

```diff
@@ -70,15 +70,23 @@ module.exports.test = (uiTestCtx) => {
           .click('a[href="/settings/licenses"]')
           .wait('a[href="/settings/licenses/pick-lists"]')
           .click('a[href="/settings/licenses/pick-lists"]')
-          .waitUntilNetworkIdle(1000)
-          .wait('#clickable-edit-pick-lists-0')
-          .click('#clickable-edit-pick-lists-0')
-          .wait('input[name="items[0].desc"]')
-          .type('input[name="items[0].desc"]', editText)
-          .wait('#clickable-save-pick-lists-0')
-          .click('#clickable-save-pick-lists-0')
-          .waitUntilNetworkIdle(1000)
-          .then(done)
+          .waitUntilNetworkIdle(2000)
+          .evaluate(_pickList => {
+            const rowNumber = [...document.querySelectorAll('#editList-pick-lists [role="row"]')].map(e => e.textContent).findIndex(i => i.indexOf(_pickList) >= 0) - 1; // figure out the row number
```

Comment:
Break long lines like this across multiple lines for ease of reading. Eg:
```
return [...]
  .map()
  .findIndex();
```

Reply:
Also, the `- 1` is a bit magic-numbery. _I_ know why it's there, but it took me a few seconds and isn't obvious. Is there a way to use a property of the matching node itself, perhaps an `id`, and return that for use in the `.then`?

Reply:
i don't get it 
do you have a hint?

Reply:
I tried to come up with a nice clean way but it just makes the code more complicated. Maybe just put a comment up above the line saying why you're doing `- 1` (to account for the row header).